import/index_java/src/org/vufind/index/GeoTools.java

```diff
@@ -314,15 +402,15 @@ public boolean validateValues(Record record, Double west, Double east, Double no
     */
    public boolean validateExtent(Record record, Double west, Double east, Double north, Double south) {
     if (west > 180.0 || west < -180.0 || east > 180.0 || east < -180.0) {
-        ControlField recID = (ControlField) record.getVariableField("001");
-        String recNum = recID.getData();
-        logger.error("Record ID: " + recNum.trim() + " - Coordinates exceed map extent.");
+        String msgError = "Coordinates exceed map extent.";
+        HashMap<Character, String> coords = buildCoordinateHashMap(west, east, north, south);
+        logErrorMessage(record, coords, msgError);
         return false;
     }
     if (north > 90.0 || north < -90.0 || south > 90.0 || south < -90.0) {
```

Comment:
Do we need two separate if statements here? It looks like the result of the condition is identical for both blocks. How about one big:

<pre>
if (west > 180.0 || west < -180.0 || east > 180.0 || east < -180.0
    || north > 90.0 || north < -90.0 || south > 90.0 || south < -90.0
) {
</pre>

instead? Using that formatting still keeps the two sets of conditions on their own lines, but it's a little more concise.

Reply:
That's definitely more concise. Thanks!